mtp_cashbook/templates/mtp_auth/login.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+
+{% block content %}
+{% if form.errors and not form.non_field_errors %}
+<p class="errornote">
+{% if form.errors.items|length == 1 %}Please correct the error below.{% else %}Please correct the errors below{% endif %}
```

Comment:
Well, we're doing it elsewhere, and we'll need to support Welsh at some point so if it's easy-ish, we should do it.